pypga/modules/migen/spimachine.py

```diff
@@ -144,11 +144,29 @@ def __init__(self, data_width=32, div_width=8):
                 self.cs_next.eq(0),#1
                 If(self.load,
                     cg.count.eq(1),
-                    reg.load.eq(1),
+                    NextState("START1")
+                )
+        )
+        fsm.act("START1",
+                self.cs_next.eq(1),
+                cg.count.eq(1),
+                cg.extend.eq(1),
+                self.clk_next.eq(0),
+                If(cg.done,
+                   NextState("START2")
```

Comment:
Hacky way for having a longer delay time between CS going low and the first clock cycle.